### PR TITLE
Updated the UI of the home screen

### LINKED ISSUE #73: [FEATURE] Enhancing the UI of the cards in the home screen
### Description

The UI of cards in home screen looks simplistic and basic. I'd enhance the UI capabilities of cards in this section, so that they become more eye-catching and better in-sync with the rest of the app's UI.

### Screenshots

Screenshot_20230730_081356

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by jaivsh**
## Description

I've updated the UI associated with the home screen to make the poem cards look more attractive and in-sync with the overall UI of the application. 

## Screenshots
Screenshot_20230806_153330

### CODE REVIEW

**Comment by jaivsh** [comment]
@bhattabhi013 please review this PR

**Comment by bhattabhi013** [comment]
Hi @jaivsh,
Can you please remove the bg-color of header?

**Comment by jaivsh** [comment]
> Hi @jaivsh,
> Can you please remove the bg-color of header?

Hi @bhattabhi013, I've removed the background color of header. Please can u merge it now?

File: lib/Home/screens.dart [hunk 1]
```diff
@@ -93,8 +93,24 @@ class GifCard extends StatelessWidget {
   Widget build(BuildContext context) {
     final mediaquery = MediaQuery.of(context);
 
-    return Container(
-      color: Theme.of(context).colorScheme.background,
+    return Material(elevation: 0,child: Container(
+
```
Comment (bhattabhi013) [inline lib/Home/screens.dart:97]: Hi,
Why have you used Material here?
You could've used the `Card` widget instead.

**Review by bhattabhi013 — CHANGES_REQUESTED** [review]
Please look at the comments

File: lib/Home/screens.dart (hunk 1 above)
Comment (jaivsh) [reply lib/Home/screens.dart]: Actually initially I was proiding the elevation through the material widget, but it was not possible to provide a box-elevation there. So, i basically did this via container decoration itself, and did the material elevation to 0. Nevertheless, I've removed the material widget @bhattabhi013 so can u please review it now?

**Comment by jaivsh** [comment]
> Please look at the comments

@bhattabhi013 resolved it: removed the redundant material widget.

**Review by bhattabhi013 — APPROVED** [review]

**Comment by bhattabhi013** [comment]
LGTM🚀

**Pull request merged**